tests/test_wave_01.py

```diff
@@ -158,13 +159,13 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
-    
-    raise Exception("Test needs to be completed.")
+    assert any(movie["title"] == MOVIE_TITLE_1 for movie in updated_data["watched"])
```

Comment:
Nice using the `any` function to check if the generator contains `True` after looping through  to see if a movie's title matches `MOVIE_TITLE_1`. I think that it might be more safeguarded to check if the entire movie is present in a list rather than title though. Mainly so that it makes sure that a that entire movie dictionary is present.